src/core/tree_methods.ts

```diff
@@ -0,0 +1,84 @@
+import * as _ from 'lodash'
+
+export class TreeNode<T> {
+  content: T;
```

Comment:
I think "payload" is better, I've just realized we're using "content" in resource types.

Reply:
bump

Reply:
`data` would also work

Reply:
sry didnt notice, consider it changed